src/components/Input/Input.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components";
+
+const Input = (props) => {
+  return <StyledInput {...props} />;
+};
+
+const StyledInput = styled.input`
+  padding: 4px;
+  font-size: 32px;
+  font-weight: 500;
+  color: ${(props) => props.theme.woowagreen};
```

Comment:
```suggestion
  color: ${({theme}) => theme.woowagreen};
```

어떤 폴더에서는 비구조화를, 현재는 비구조화를 적용하지 않았는데요~
통일성있게 가져가면 좋을 것 같습니다!

Reply:
피드백 감사합니다 분해할당하는 것으로 통일했어요